src/containers/challenge/ChallengePageQueries.graphql

```diff
@@ -74,13 +74,13 @@ fragment ChallengeProfile on Challenge {
         }
```

Comment:
just moved this higher, closer to memberOrganizations for clarity, did the same in the Opportunities one